tools/profiling/templates/stopwatch.tpl

```diff
@@ -63,7 +63,11 @@
           </td>
           <td data-value="{$data['location']}">
             <a href="javascript:void(0);" onclick="$('#callstack_{$callstack_md5}').toggle();">{$data['location']}</a>
-            <div id="callstack_{$callstack_md5}" style="display:none">{implode('<br>', $data['stack'])}</div>
+            <div id="callstack_{$callstack_md5}" style="display:none">
+              {foreach $data['stack'] as $stack}
```

Comment:
No need to implode with nofilter when you can use foreach :)

Reply:
You get a trailing `<br>` like this